client/src/Components/UserSignUp.js

```diff
@@ -135,24 +138,22 @@ import {Redirect} from 'react-router-dom';
                 type="password"
                 id="password"
                 onClick={e => setPassword(e.target.value)}
-
               />
             </Grid>
-            
           </Grid>
           <Button
             type="submit"
             fullWidth
             variant="contained"
             color="primary"
             className={classes.submit}
-            onClick={e => setValue(e.target.value)}
+            // onClick={e => setValue(e.target.value)}
```

Comment:
This is the submit button, this should be passing the handleSubmit event instead of setValue like it was.